README.md

```diff
@@ -27,9 +27,11 @@ The module allows you to manage NetScaler nodes and pool configuration through P
 
 ###Beginning with netscaler
 
-Before you can use the netscaler module, you must create a proxy system able to run `puppet device`.  In order to do so, you will have a Puppet master and a Puppet agent as usual, and the agent will be the "proxy system" for the puppet device subcommand.
+Before you can use the netscaler module, you must create a proxy system able to run `puppet device`.  In order to do so, you will have a Puppet master and a Puppet agent as usual, and the agent will be the "proxy system" for the `puppet device` subcommand.
 
-**NOTE: puppet device was designed to interface with the admin interface to run commands to implement type/provider. puppet device allows you to write a resource to talk to the device (ask Hunter about specific implementation) translates parts of the module (providers and such)into network calls
+**NOTE: `puppet device` was designed to interact with admininistrative interfaces in order to run commands that implement types and providers. `puppet device` allows you to write a resource to talk to the device and translates the module into network calls.**
```

Comment:
Yeah, I know. But given Adam's issues and the lack of documentation on puppet device in general, I want something there to help guide the user's thinking about and approach to `puppet device`. Particularly since this is a Puppet Enterprise module.